Loop Status Extension/StatusViewController.swift

```diff
@@ -92,7 +92,8 @@ class StatusViewController: UIViewController, NCWidgetProviding {
         observationEnabled: false,
         insulinModel: defaults?.insulinModelSettings?.model,
         basalProfile: defaults?.basalRateSchedule,
-        insulinSensitivitySchedule: defaults?.insulinSensitivitySchedule
+        insulinSensitivitySchedule: defaults?.insulinSensitivitySchedule,
+        provenanceIdentifier: HKSource.default().bundleIdentifier
```

Comment:
Does this end up with different provenanceIdentifiers for Loop and the extension? Should we be using something like the app group identifier?

Reply:
This mirrors the behavior of what HealthKit would do if you added entries from the app, extension, watch extension, etc.  In our case, we don't actually write any data *real* data in any place other than the main app. (Yes, we do write "shadow" data in the watch extension, for example, in order to temporarily persist data for display only. For example, when the user enters carbs on the watch it goes through the main app, is written to the main app cache and HealthKit, and then, later, that data sent back to the watch and written to the watch cache as a "shadow copy" of the data in the main app. It is used for display purposes only.)